shell/platform/darwin/ios/framework/Source/FlutterEngineTest.mm

```diff
@@ -79,4 +79,18 @@ - (void)testNotifyPluginOfDealloc {
   OCMVerify([plugin detachFromEngineForRegistrar:[OCMArg any]]);
 }
 
+// TODO(justinmc): Or is this the way to test platform channel messages? How can
+// I see if it was handled?
+/*
+- (void)testHasStrings {
+  id project = OCMClassMock([FlutterDartProject class]);
+  FlutterEngine* engine = [[FlutterEngine alloc] initWithName:@"foobar" project:project];
```

Comment:
There should be a way to create a mock clipboard, and pass it into the mock engine/plugin, though not clear on implementation details